Makefile.buildtests

```diff
@@ -70,6 +70,20 @@ buildtest:
 			fi; \
 			break; \
 		done; \
+		LOG=$$(env -i \
```

Comment:
`${LOG}` is not printed or used afterwards, but ignoring the output should be fine.